skcosmo/linear_model/_ridge.py

```diff
@@ -8,7 +8,23 @@
 
 
 class RidgeRegression2FoldCV(MultiOutputMixin, RegressorMixin):
-    """Ridge regression with an efficient 2-fold cross validation method using the SVD solver.
+    r"""Ridge regression with an efficient 2-fold cross validation method using the SVD solver.
+    Minimizes the objective function:
+
+        .. math:: \|y - Xw\|^2_2 + \alpha \|w\|^2_2, % # noqa: W605
+
+    while the alpha value is determined with a 2-fold cross validation from a list of
+    alhpa values. It is more efficient than doing a 2-fold cross validation using
+    :obj:`sklearn.linear_model.RidgeCV`.
+    The advantage over :obj:`sklearn.linear_model.RidgeCV` using LOOCV needs to be
```

Comment:
What's LOOCV?

Reply:
leave-one-out cross-validation, added it to doc, also added a reference with the efficient procedure for LOOCV because it is not obivious